app/cors.py

```diff
@@ -0,0 +1,64 @@
+from datetime import timedelta
+from flask import make_response, request, current_app
+from functools import update_wrapper
+
+
+def crossdomain(origin=None, methods=None, headers=None,
+                max_age=21600, attach_to_all=True,
+                automatic_options=True):
+    """
+    Flask view decorator to add CORS headers to a view. Keyword arguments:
+
+    `methods` - Optionally a list of methods that are allowed for this view.
+        If not provided it will allow all methods that are implemented.
+    `headers` - Optionally a list of headers that are allowed for this request.
+    `origin` - '*' to allow all origins, otherwise a string with a URL or a
+        list of URLs that might access the resource.
+    `max_age` - The number of seconds as integer or timedelta object for which
+        the preflighted request is valid.
+    `attach_to_all` - True if the decorator should add the access control
+        headers to all HTTP methods or False if it should only add them to
+        OPTIONS responses.
+    `automatic_options` - If enabled the decorator will use the default Flas
+        OPTIONS response and attach the headers there, otherwise the view
+        function will be called to generate an appropriate response.
+
+    Adapted from http://flask.pocoo.org/snippets/56/
```

Comment:
It looks like https://pypi.python.org/pypi/Flask-Cors/ is based on the same snippet, could we use that instead? Less code to write = win